flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/AdaptedRestartPipelinedRegionStrategyNGConcurrentFailoverTest.java

```diff
@@ -250,37 +253,38 @@ public void testSkipFailoverIfExecutionStateIsNotRunning() throws Exception {
 	 */
 	private ExecutionGraph createExecutionGraph() throws Exception {
 
-		final JobInformation jobInformation = new DummyJobInformation(TEST_JOB_ID, "test job");
-		final SimpleSlotProvider slotProvider = new SimpleSlotProvider(TEST_JOB_ID, DEFAULT_PARALLELISM);
-
-		final Time timeout = Time.seconds(10L);
-		final ExecutionGraph graph = new ExecutionGraph(
-			jobInformation,
-			TestingUtils.defaultExecutor(),
-			TestingUtils.defaultExecutor(),
-			timeout,
-			manuallyTriggeredRestartStrategy,
-			TestAdaptedRestartPipelinedRegionStrategyNG::new,
-			slotProvider,
-			getClass().getClassLoader(),
-			VoidBlobWriter.getInstance(),
-			timeout);
-
-		JobVertex v1 = new JobVertex("vertex1");
+		final JobVertex v1 = new JobVertex("vertex1");
 		v1.setInvokableClass(NoOpInvokable.class);
 		v1.setParallelism(DEFAULT_PARALLELISM);
 
-		JobVertex v2 = new JobVertex("vertex2");
+		final JobVertex v2 = new JobVertex("vertex2");
 		v2.setInvokableClass(NoOpInvokable.class);
 		v2.setParallelism(DEFAULT_PARALLELISM);
 
 		v2.connectNewDataSetAsInput(v1, DistributionPattern.ALL_TO_ALL, ResultPartitionType.BLOCKING);
 
-		JobGraph jg = new JobGraph(TEST_JOB_ID, "testjob", v1, v2);
-		graph.attachJobGraph(jg.getVerticesSortedTopologicallyFromSources());
+		final JobGraph jg = new JobGraph(TEST_JOB_ID, "testjob", v1, v2);
+
+		final SimpleSlotProvider slotProvider = new SimpleSlotProvider(TEST_JOB_ID, DEFAULT_PARALLELISM);
+
+		final Configuration jmConfig = new Configuration();
+		jmConfig.setBoolean(JobManagerOptions.FORCE_PARTITION_RELEASE_ON_CONSUMPTION, false);
```

Comment:
this should no longer be necessary; `false` is now the default.

Reply:
Yes.